### ISSUE TITLE
use WERCKER_HUGO_DISABLE_PYGMENTS incorrectly?

### ISSUE THREAD

**Issue by foreverfaint** [ArjenSchwarz/wercker-step-hugo-build#16]
Hi, 

"DISABLE != false"  means "DISABLE == true", and then install pygments, is it a little weird?

if $WERCKER_HUGO_DISABLE_PYGMENTS != false; then
    install_pygments
fi

**Comment by ArjenSchwarz** [comment]
That's a fair point @foreverfaint, thanks. I'll have a look at the logic here and clean it up.

**Comment by sdebruyn** [comment]
I wrote it like that because
1. The default value is false so I just check if the default value is set or not.
2. My bash skills are not that great...

**Comment by ArjenSchwarz** [comment]
@foreverfaint this is fixed in the latest version, thanks for catching it.

@SamuelDebruyn don't worry about it, your contributions are still highly appreciated and I didn't catch it either. The main issue is that the logic was wrong, the code was set to install pigments when disable_pygments = true. Additionally, I found that the true/false values weren't properly handled (an issue with the way the values are parsed), so I've made some fixes to that as well. I'm planning a restructuring of the code to make things like this work better and to allow it to be tested so we can prevent it from happening.
